runtime/autoload/health/provider.vim

```diff
@@ -551,7 +551,10 @@ function! s:check_virtualenv() abort
     let msg .= "\nSo invoking Python may lead to unexpected results."
     call health#report_warn(msg, keys(hints))
   else
-    call health#report_ok('$VIRTUAL_ENV provides :python, :python3, et al.')
+    call health#report_info(msg)
+    call health#report_info('Python version: '
+      \.system('python -c "import platform, sys; sys.stdout.write(platform.python_version())"'))
```

Comment:
wont that fail if you don't have python installed ?

Reply:
This line of code shouldn't be reached unless a `python` executable was found inside the configured `$VIRTUAL_ENV`, so it shouldn't fail. If you can think of ways it could be reached without a `python` command on the path, then please let me know, it would be a logic error in the check :)